Makefile

```diff
@@ -76,14 +76,16 @@ pyenv:
 
 secrets:
 	@[ "${SECRET_SA_ACCOUNT_NAME}" ] || ( echo ">> SECRET_SA_ACCOUNT_NAME is not set"; exit 1 )
-	rm -rf secrets
+	@[ "${SECRETS}" ] || ( echo ">> SECRETS is not set"; exit 1 )
+	rm -rf ${SECRETS}
 	az storage blob download -n secrets.tar.gz -c secrets -f secrets.tar.gz --account-name ${SECRET_SA_ACCOUNT_NAME} >/dev/null
-	tar -xzf secrets.tar.gz
+	mkdir ${SECRETS} && tar -xzf secrets.tar.gz -C ${SECRETS}
```

Comment:
I dont see any issues if when secret is updated old one (public cloud or fairfax) is removed. We should not keep all of them on the same place stacking and rotting.

Reply:
@ihcsim There would be just one additional directory and one additional file added at the repo base dir if you used the suggested directory naming scheme, which isn't that bad. Technically you could personally choose to set your `SECRETS` variable to `secrets/public` instead of `secrets.public` or whatever -- I don't think I've come across any code that would be broken if you wanted to clean up your local setup like that. The `gitignore` file will exclude anything matching `/secrets*`.

I did originally think of using symlinks for secrets changing, with a little script, but it was probably too complicated, despite touching fewer files: https://github.com/Azure/ARO-RP/pull/1268/files

@mjudeikis You're suggesting we have a single folder of secrets at a time, and run `make secrets` after switching subscription/cloud each time? I suppose that would work, too.

I'm pretty indifferent on the exact approach; this PR was @jim-minter's suggestion, so I'll wait to see what he says before changing anything again.

Reply:
> There would be just one additional directory and one additional file added at the repo base dir 

The number grows with the number of subscriptions.

Reply:
> > There would be just one additional directory and one additional file added at the repo base dir
> 
> The number grows with the number of subscriptions.

Then `export SECRETS=secrets/yournamehere` and group them in one folder like I suggested above.

Reply:
@mjudeikis Not a bad idea. The current multi-folder approach stems from the continuation of what the folder structure would look like after following the steps to [prepare a shared RP dev env](https://github.com/Azure/ARO-RP/blob/8801190e8be8540b23c77cff6464bbc1b9b28df4/docs/prepare-a-shared-rp-development-environment.md).